src/processing/code.rs

```diff
@@ -535,12 +537,14 @@ impl FromStr for SnippetLanguage {
             "sql" => Sql,
             "svelte" => Svelte,
             "swift" => Swift,
+            "tcl" => Tcl,
             "terraform" => Terraform,
             "toml" => Toml,
             "typescript" | "ts" => TypeScript,
             "typst" => Typst,
             "xml" => Xml,
             "yaml" => Yaml,
+            "v" => Verilog,
```

Comment:
This is the string you'll use in markdown to identify the snippet. I think this needs to be `verilog` no? At least github's markdown uses verilog. `v` is another language.